src/components/UsernameEntry/UsernameEntry.js

```diff
@@ -4,6 +4,21 @@ import './UsernameEntry.css';
 import ListSelection from '../listSelection/ListSelection';
 
 class UsernameEntry extends React.Component {
+  state = {
+    showConfirmButton: false,
+    selectedName: null
+  };
+
+  promptToConfirm = e => {
+    this.setState({showConfirmButton: true, selectedName: e.name});
+  };
+
+  deselect = () => {
+    if (this.state.showConfirmButton) {
+      this.setState({showConfirmButton: false, selectedName: null});
```

Comment:
Just thinking, do we need two bits of state for this or can we just use whether we've selected a user?

e.g, something like...

```
state = {
  selectedUser: null
}

handleTouchMove() {
  this.setState({ selectedUser: null })
}

{this.state.selectedUser && (
  <div>...Next...</div>
)}

<ListSelection
  selected={this.state.selectedUser.name}
/>
```